cypress.config.js

```diff
@@ -0,0 +1,20 @@
+const { defineConfig } = require('cypress');
+
+module.exports = defineConfig({
+  e2e: {
+    baseUrl: 'http://localhost:3000',
+    supportFile: false,
+    specPattern: 'cypress/integration/**/*.spec.{js,ts,jsx,tsx}',
+  },
+  video: true,
+  viewportHeight: 1920,
```

Comment:
The configuration option 'video' expects a boolean value indicating whether to record video of the test runs. If you want to enable video recording, it should be 'video: true'. If this line is meant to enable video recording, it is correct, but if it's meant to specify video options, it should be nested under a 'videoOptions' object.